kde-frameworks/kdelibs4support/kdelibs4support-5.109.0-r1.ebuild

```diff
@@ -88,7 +88,11 @@ src_prepare() {
 	ecm_src_prepare
 
 	if ! use handbook; then
-		sed -e "/kdoctools_install/ s/^/#DONT/" -i CMakeLists.txt || die
+		# kdelibs4support has a weird implementation.
+		# It includes KF5DocTools dependency unconditionally into cmake macros,
+		# causing some other packages (like plasma-desktop) build to crash if KDocTools is not installed.
+		sed -e "/find_dependency(KF5DocTools/ s/^/# /" -i KF5KDELibs4SupportConfig.cmake.in || die
+		sed -e '/include("${KF5DocTools/ s/^/# /' -i cmake/modules/KDE4Macros.cmake || die
```

Comment:
Be careful here. Previously this was only dropping installation of translated kdelibs4support handbook files. You're dropping that, though, so they will be installed unconditionally?

Then, your change is meddling with cmake macros provided by kdelibs4support for use by reverse dependencies. It is breaking `KDE4_CREATE_HANDBOOK` later in `KDE4Macros.cmake` and consequently any reverse dependency relying on kdelibs4support providing this macro. That's precisely why `=kde-frameworks/kdoctools-${PVCUT}*:5` is found in `RDEPEND` in this ebuild.

Reply:
It is not pointless at all. It avoids `khelpcenter`, with it 'khtml', and last but not least all the installed handbook files.

Did you check the size of kdoctools? We wouldn't go through all the trouble just for that:
```
$ equery s kdoctools
 * kde-frameworks/kdoctools-5.109.0
         Total files : 1245
         Total size  : 6.83 MiB
```

Reply:
I can understand the sentiment, but I need to be very clear here: We are not making exercises in dependency purity. What upstream did not fix in time (plasma-desktop:5 will never not depend kdelibs4support) we can not make up for. We must not break potentially countless revdeps depending on the macro.